int8 x
int8 y
---
bool complete_status
int8 reward
float64 distance_to_go
---
float64 distance_moved
